### ISSUE TITLE
warning: lib32-tcl: local (8.6.4-2) is newer than archlinuxcn (8.6.4-1.1)warning: lib32-tcl: local (8.6.4-2) is newer than archlinuxcn (8.6.4-1.1)

### ISSUE THREAD

**Issue by colinkeenan** [archlinuxcn/repo#187]
warning: lib32-tcl: local (8.6.4-2) is newer than archlinuxcn (8.6.4-1.1)

For some reason, http://repo.archlinuxcn.org/x86_64/lib32-tcl-8.6.4-1.1-x86_64.pkg.tar.xz was uploaded a few days ago even though https://github.com/archlinuxcn/repo/blob/master/lib32-tcl/PKGBUILD still shows the version at 8.6.4-2. 

This "new" version is numbered lower than the the "old" version. What does package build "1.1" even mean? The new version should either be "8.6.4-3" (package build 3 of the package version 8.6.4) or "8.6.4.1-1" (package build 1 of the package version 8.6.4.1) - or maybe "8.7.1-1" . . . 

It makes no sense for there to be a "1.1" package build of a package at version "8.6.4".

**Comment by lilydjwg** [comment]
The PKGBUILD in this repo does not necessarily match the packaged version. It seems that @yuyichao updated the 1.1 release some days ago.

**Comment by colinkeenan** [comment]
OK, but the problem is that the release version (1.1) makes no sense. It should be a whole number. It just represents how many times the same version of the application had been packaged for archlinuxcn. Even if he wanted to use a .1 for some reason, it would be 2.1, not 1.1 because the previous release was 2. How do you go from 2 to 1.1? It makes pacman say it's an older version than what's already installed. I was thinking maybe it was a typo and the application version has actually changed instead of staying at 8.6.4. If it's a new application version, then it makes sense to go back to a package release version of 1 (but still, not 1.1).

Bottom line, package release number 1.1 makes no sense and this was a mistake that should be corrected.

**Comment by yuyichao** [comment]
No I did not do that, I could repackage a `-2` and update it later though.

@colinkeenan I don't think you need to explain why it is bad to go from 2 to 1.1.

**Comment by lilydjwg** [comment]
Then who's done that....

**Comment by colinkeenan** [comment]
It should be `-3` now if it is the same application version.

**Comment by yuyichao** [comment]
> Then who's done that....

No idea. I didn't log in to the build server this month until today and I also don't have any script to do that automatically. Neither of the clone of the repo I have on both the build server and locally has a 1.1 release.

**Comment by farseerfc** [comment]
Might be OT, but I think that's why we should sign our uploaded packages.

**Comment by lilydjwg** [comment]
Or at least set PACKAGER to a proper value.
